### DISCUSSION TITLE
Adding Elk Web client to instance

### DISCUSSION THREAD

**Discussion by kylereddoch** [iOSDevSpace/iosdevspace#11] [Ideas]
What does the community think about us adding the [Elk web client](https://github.com/elk-zone/elk) to our instance?

**Comment by heyjaywilson** [comment]
Questions:
- Does it have to be hosted anywhere?
- Is there an extra cost to adding it?

**Reply by kylereddoch** [reply]
Yes, it will need to be hosted somewhere as this will be a subdomain (something like elk.iosdev.space). With that, there will be a cost but not like masto.host where the cost increases due to the instance growth. It could be hosted elsewhere and where ever you have the domain name (iosdev.space) registered is where you would create the subdomain.
